lecture_3/03_project/calculator/internal/engine/publisher/betPublisher.go

```diff
@@ -0,0 +1,11 @@
+package publisher
+
+import (
+	"context"
+
+	rabbitmqmodels "github.com/superbet-group/code-cadets-2021/lecture_3/03_project/calculator/internal/infrastructure/rabbitmq/models"
+)
+
+type BetPublisher interface {
```

Comment:
Consider renaming this to `BetCalculatedPublisher` as it is publishing `BetCalculated` messages.

Note I've suggested renaming `domainmodels.BetCalculated` to `Bet` and `storagemodels.BetCalculated` to `Bet`, but in this case `BetCalculated` is the proper naming because it is a data model which should be produced to `bets-calculated` queue.